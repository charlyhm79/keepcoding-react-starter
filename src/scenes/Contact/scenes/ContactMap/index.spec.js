import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ContactMap from './';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<ContactMap />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('Matchs snapshot', () => {
  const tree = renderer.create(<ContactMap />).toJSON();
  expect(tree).toMatchSnapshot();
});
