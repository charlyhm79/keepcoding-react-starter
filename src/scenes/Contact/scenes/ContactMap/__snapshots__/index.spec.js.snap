// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matchs snapshot 1`] = `
<img
  alt="contact map"
  className="sc-bdVaJa UoSJW"
  src="http://is5.mzstatic.com/image/thumb/Purple30/v4/1f/42/cb/1f42cbf2-1d83-a356-00db-8edcc0598ff2/source/552x414bb.jpg"
/>
`;
